example/master.js

```diff
@@ -4,6 +4,10 @@ import payload from "./payload.json";
 
 document.addEventListener("DOMContentLoaded", async() => {
   const secureDataSet = new NodeSecureDataSet();
+  // const secureDataSet = new NodeSecureDataSet({
```

Comment:
Forgotten comment?

Reply:
No, i wanted to keep it so people are aware they can test the example with them.